src/datepicker.ts

```diff
@@ -901,15 +901,18 @@ export class Datepicker extends LitElement {
                       'day--today': +todayDate === curTime,
                       'day--focused': !disabled && isCurrentDate,
                     })}"
-                    part="calendar-day"
+                    .part="calendar-day${+todayDate === curTime ? ' calendar-today' : ''}"
```

Comment:
why using `.part` instead of `part`?

Reply:
It was a mistake of the first try. Fixed.